src/createCycleMiddleware.js

```diff
@@ -1,4 +1,5 @@
 import xs from 'xstream'
+import {adapt} from '@cycle/run/lib/adapt'
```

Comment:
Should we use `adapt` also on https://github.com/cyclejs-community/redux-cycles/blob/master/src/combineCycles.js#L17 ?